packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -264,6 +258,7 @@ const VirtualListBaseFactory = (type) => {
 		nodeIndexToBeFocused = null
 		preservedIndex = null
 		restoreLastFocused = false
+		lastConfiguredSpotlightId = null
```

Comment:
How about `prevSpotlightId` as you did in `Scrollable` instead of `lastConfiguredSpotlightId`?